src/components/videos-you-might-like/index.js

```diff
@@ -15,20 +16,68 @@ const VideosYouMightLike = () => {
   const [videoPlayers, setVideoPlayers] = useState({});
   const [currentVideoId, setCurrentVideoId] = useState(null);
 
-  const onReady = function (videoId, event) {
+  const onReady = (videoId, event) => {
     setVideoPlayers({ ...videoPlayers, [videoId]: event.target });
   };
 
-  const stopAll = function () {
-    for (const videoPlayer of Object.values(videoPlayers)) {
-      videoPlayer.stopVideo();
-    }
+  const stopAll = () => {
+    document.querySelectorAll(".ytIFrame").forEach((iframe) => {
+      iframe.contentWindow.postMessage(
+        JSON.stringify({ event: "command", func: "stopVideo" }),
+        "*"
+      );
+    });
   };
 
-  const play = function (videoId) {
+  const play = (videoId) => {
     setCurrentVideoId(videoId);
     stopAll();
-    videoPlayers[videoId].playVideo();
+    videoPlayers[videoId]?.playVideo();
+  };
+
+  const youtubeVideos = () => {
+    return videos.map((video) => (
+      <YoutubeVideo
+        key={video.id}
+        videoId={video.id}
+        className={cx("h-[80%]", video.id !== currentVideoId && "hidden")}
```

Comment:
There's a `h-4/5` class that you can use:
```suggestion
        className={cx("h-4/5", video.id !== currentVideoId && "hidden")}
```